### DISCUSSION TITLE
MacOS 10.14 and older builds possible?

### DISCUSSION THREAD

**Discussion by joshhh** [utmapp/UTM#3993] [UTM Help (macOS)] [answered]
Is it possible to build UTM for MacOS 10.14 and older OS?

**Comment by osy** [comment, marked as answer]
No, it's built on top of Apple APIs only introduced in macOS 11

**Reply by joshhh** [reply]
Linux build also relies on MacOS 11 APIs?

**Reply by osy** [reply]
There is no Linux build obviously.
